### ISSUE TITLE
Bed Finder Talent not working

### ISSUE THREAD

**Issue by Kavaltheone** [DashieDev/DoggyTalentsNext#71]
Hello.
DTN Version 1.18.10, Fabric, MC 1.20.1
Bed Finder talent on wiki shows yellow outline on bed. But it's not working. Example shows my bed and my dog's bed, none of them outlined, skill level maxed.
Oh, and thanks for fixing dog on head in this version ^_^

https://github.com/DashieDev/DoggyTalentsNext/assets/12968480/6779a3a5-5ffa-4f37-a68f-d87bd12f6108

**Comment by DashieDev** [comment]
Yeah currently bed finder seems to not works occaionsally on both version... 
I will address this in the future.
